### PR TITLE
MEX-468 Add "referred by" question to the top of consult note

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/MEX-468

### CODE REVIEW

**Comment by brandones** [comment]
Screenshot from 2019-11-20 14-40-50

**Comment by brandones** [comment]
The text will make more sense in Spanish.

File: api/src/main/resources/messages.properties [hunk 1]
```diff
@@ -643,6 +643,13 @@ pihcore.visitNote.plan.approximateReturnVisitDate=Approximate next appointment d
 pihcore.visitNote.ncdInitial=NCD 
 pihcore.visitNote.ncdFollowup=NCD
 
+pihcore.visitNote.referredBy=Referred by
+pihcore.visitNote.hasAppointment=Has appointment
+pihcore.visitNote.walkIn=Walk-in
+pihcore.visitNote.activeCasefinding=Active casefinding
+pihcore.visitNote.chw=CHW
```
Comment (lnball) [inline api/src/main/resources/messages.properties]: This is fine.  One suggestion -- no need to add visitNote to the message string -- especially not for CHW.  If it's a commonly used string, just use pihcore.chw=CHW

**Comment by brandones** [comment]
Nice, thanks @lnball !

**Pull request merged**